docs/index.bs

```diff
@@ -3067,6 +3107,8 @@ spec: storage; urlPrefix: https://storage.spec.whatwg.org/
       1. Assert: |request|'s [=request/destination=] is not "<code>serviceworker</code>".
       1. If |request|'s [=request/destination=] is either "<code>embed</code>" or "<code>object</code>", then:
           1. Return null.
+      1. Else if |registration|'s <a>active worker</a>'s [=service worker/list of router rules=] is set:
+          1. If running [=GetRouterSource=] algorithm with <a>active worker</a> and |request| returns "network", return null.
```

Comment:
```suggestion
          1. If running [=GetRouterSource=] with |registration|'s <a>active worker</a> and |request| returns "network", return null.
```

Reply:
I guess the goal for this is clarifying |registration|.  I kept algorithm as-is.